### DISCUSSION TITLE
Valour DB reset

### DISCUSSION THREAD

**Discussion by lixsady** [Valour-Software/Valour#969] [Q&A] [answered]
When valour was first starting up over the spookvooper community i created a few accounts that i can no longer access, is this user error or was their a database reset from then and now. sorry if im just putting in my password or something wrong.

**Comment by Ryguy6001** [comment, marked as answer]
There was a DB reset awhile ago you'll need to make a new account.

**Reply by lixsady** [reply]
alright, good to know
